app/controllers/listener/lines_controller.rb

```diff
@@ -6,13 +6,17 @@ def check
       reply_token = event['replyToken']
       rumor       = event['message']['text']
 
-      ReplyWorker.perform_async(reply_token, rumor) if rumor
+      ReplyWorker.perform_async(reply_token, rumor) if rumor and probably_forwarded?(rumor)
```

Comment:
cool, prefer

```suggestion
       ReplyWorker.perform_async(reply_token, rumor) if rumor && probably_forwarded?(rumor)
```

Reply:
or, maybe this one would be better

```
ReplyWorker.perform_async(reply_token, rumor) if forwardable?(rumor)

def forwardable?(rumor)
  return unless rumor
  rumor.size > 20
end
```

wdyt?

Reply:
Hmm okay